src/lambda_codebase/account_processing/get_account_regions.py

```diff
@@ -16,6 +16,24 @@
 ADF_ROLE_NAME = os.getenv("ADF_ROLE_NAME")
 AWS_PARTITION = os.getenv("AWS_PARTITION")
 
+def get_default_regions_for_account(ec2_client):
+    default_regions = [
+    region["RegionName"]
```

Comment:
Could you fix the indentation in this function. I think lines 21 to 33 need to get indented once more.